packages/samples/testserver/body-time/body-time.cadl

```diff
@@ -1,6 +1,7 @@
 import "@cadl-lang/rest";
 
 using Cadl.Http;
+using Cadl.Rest;
```

Comment:
why is this now needed for those?

Reply:
The `@route` decorator has been moved to `Cadl.Rest` so that it can now be used without pulling in `Cadl.Http` when not needed.  We can definitely discuss this in a design meeting

Reply:
hhm so if you want to use http without rest the problem is inverted?

Reply:
Sort of, yeah. I've been considering getting rid of Cadl.Http and merging everything back into Cadl.Rest because there is inevitable overlap between the two

Reply:
Oh don't do this!  The feedback I have from the MS Graph team is that they want to use the Http library only -- not rest.  And they want the Http library to be a "first class" library ... not underneath Rest.

The Http library should be for things that are part of the formal Http spec (RFC 7231 and friends).  "Rest" means different things to different people/teams so we need the separation.

Reply:
OK, that's good feedback.  The problem we end up with in that case is HTTP decorators leaking through to the REST layer, but that's sort of expected, I suppose.  I think I might need to move `@route` back to `Cadl.Http`.

Reply:
Moved `@route` back to `Cadl.Http`, now `Cadl.Rest` is not needed in this case.